_po4a-tools/po4a-stats.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+
+# Stats file location
+STATS_FILE="../wiki/Statistics.md"
+
+# Print yaml front matter and table title/header
+echo '---
+layout: wiki
+title: "Statistics"
+lang: "en"
+permalink: "/wiki/Statistics"
```

Comment:
I added this despite not being sure where this file can be made available.

Reply:
Yes. Adding this file to the site is probably better.

Reply:
Here's the thing though: the version displayed on the site would be the translation status of 'release'. What about if a translator wants to know about the status in 'next-release'? The stats file gets created for both branches, but the one for 'next-release' gets "lost" as it isn't published anywhere.

Reply:
Hmm yes. If we succeed to have an issue which gets updated updated automatically, this would also work. 

https://github.com/marketplace/actions/create-or-update-comment

better: https://stackoverflow.com/questions/66756072/updating-github-issues-from-github-actions to avoid needing to add unknown code.

Reply:
This is also beyond my capabilities. The stats file for 'next-release' will be in the zipped site anyway, so not entirely lost...

Reply:
Ok.